pyfixest/did/visualize.py

```diff
@@ -74,24 +77,72 @@ def panelview(
     )
     ```
     """
```

Comment:
I think the `subsamp` and `collapse_to_cohort` should start working if you move some of this code 

```python
    treatment_quilt = data.pivot(index=unit, columns=time, values=treat)
    treatment_quilt = treatment_quilt.sample(subsamp) if subsamp else treatment_quilt
```
above your new code?

Reply:
Hi @s3alfisc , i think its not as simple as that with my current code, i have just added some changes to code though:
- Add subsamp and collapse_to_cohort for outcome plot
- Adding docstring for figsize,  outcome, and type

But im thinking to pivoting my data first before plotting it so that the implementation of the subsamp and collapse_to_cohort can be the same with the apoorva's code (just like you mentioned above)

I think another things to do from my end:
- Im considering to use different data for plotting the outcome plot to have more visible control vs treatment (pre) vs treatment (post) line, what do you think? @s3alfisc 
- Adding parameter to flexibly choose which unit/group to plot --> So user can specify which group to be plotted/compare

Reply:
>     Add subsamp and collapse_to_cohort for outcome plot --> I have tested it and it works just fine
>     Adding docstring for figsize, outcome, and type

Great, thank you! 

> But im thinking to pivoting my data first before plotting it so that the implementation of the subsamp and collapse_to_cohort can be the same with the apoorva's code (just like you mentioned above) just to make it more consistent

I think this would be great, if only for the reason to keep the code base simple. 

> Adding parameter to flexibly choose which unit/group to plot --> So user can specify which group to be plotted/compare

This would be very cool I think! Would you like to add this in this or in a separate PR?

> Im considering to use different data for plotting the outcome plot to have more visible control vs treatment (pre) vs treatment (post) line, what do you think? @s3alfisc 

Here I have to admit that I am not really following 😅 Did you mean to say "colors" instead of "data" 😄

Reply:
Yeah both args now work very nicely. Cool!

Subsamp: 
![image](https://github.com/user-attachments/assets/c5d648d3-6a51-4777-839d-b98f34203804)

collapse: 
![image](https://github.com/user-attachments/assets/c5341920-52e1-4062-a26f-bbf1bcbbf452)

Maybe you can update the did notebook and update the example to `collapse_to_cohort = True`?